coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -16,11 +16,18 @@ class IgnoreResultAction(ResultAction):
 
     @staticmethod
     @enforce_signature
-    def is_applicable(result: Result, original_file_dict, file_diff_dict):
+    def is_applicable(result: Result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=[]):
```

Comment:
type annotation would be an iterable as long as the code doesn't rely on mutability; applied_actions is a continuously changing thing so a mutable arg makes sense as long as it's just only mutable within one fuction and not through many (because that adds complexity)